src/services/protobufService.ts

```diff
@@ -90,4 +91,13 @@ export class ProtobufService {
         )
     );
   }
+
+  /**
+   * List protobuf schemas
+   */
+  public async getSchemaList(): Promise<Either<ActionResponse, string[]>> {
```

Comment:
this is not the schema list, this is  a list of the types. the name is not correct